automated-comments/java/general/method_too_long.md

```diff
@@ -0,0 +1,6 @@
+At least one method in this solution is getting long.
```

Comment:
Same here: maybe add the name of the method per parameter.

Reply:
And perhaps also explain what is meant by "long" (too many arguments, arguments with large names, too much lines of code)?